### PR TITLE
Group default conflict check

### PR DESCRIPTION
**Pull request by darionyaphet**
Group default as the global resource should not created

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
I think you should change the checking in storage, if group name is `default`, don't check it exists.

**Comment by darionyaphet** [comment]
> I think you should change the checking in storage, if group name is `default`, don't check it exists.

That's not the truth You should not create the `default` group. Not Check

File: src/executor/admin/GroupExecutor.cpp [hunk 1]
```diff
@@ -14,6 +14,11 @@ namespace graph {
 folly::Future<Status> AddGroupExecutor::execute() {
     SCOPED_TIMER(&execTime_);
     auto *agNode = asNode<AddGroup>(node());
+    if (agNode->groupName() == "default") {
```
Comment (yixinglu) [inline src/executor/admin/GroupExecutor.cpp:17]: Maybe we should check whether the group name is default in parser or validator rather than executor here.

Comment (darionyaphet) [reply src/executor/admin/GroupExecutor.cpp:17]: reasonable

**Review by yixinglu — APPROVED** [review]
LGTM

File: src/validator/AdminValidator.cpp [hunk 2]
```diff
@@ -92,8 +92,12 @@ Status CreateSpaceValidator::validateImpl() {
                 } else {
                     spaceDesc_.set_isolation_level(meta::cpp2::IsolationLevel::DEFAULT);
                 }
+                break;
             }
             case SpaceOptItem::GROUP_NAME: {
+                if (item->getGroupName() == "default") {
```
Comment (panda-sheep) [inline src/validator/AdminValidator.cpp:98]: Use `folly::toLowerAscii` to deal with, and then compare ？

Comment (darionyaphet) [reply src/validator/AdminValidator.cpp:98]: Because group name case sensitive, so it's only limit the `default`

**Comment by yixinglu** [comment]
You could add some tests in feature file to test the failure error.

File: src/validator/AdminValidator.cpp (hunk 2 above)
Comment (panda-sheep) [reply src/validator/AdminValidator.cpp:98]: `default `as the group name, should ignore the case sensitive， WDYT

**Review by yixinglu — DISMISSED** [review]

**Review by yixinglu — DISMISSED** [review]

**Review by yixinglu — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]

**Pull request merged**